README.md

```diff
@@ -141,3 +141,24 @@ Dict{Symbol, Vector{Nemo.fmpq_mpoly}} with 2 entries:
 --->
 
 For more examples we refer to the documentation and the `examples` directory.
+
+## Citation
+
+If you use this package, the right way to cite is currently:
+
+```
+@article{DDP24,
+title = {Exact hierarchical reductions of dynamical models via linear transformations},
+journal = {Communications in Nonlinear Science and Numerical Simulation},
+volume = {131},
+pages = {107816},
+year = {2024},
+issn = {1007-5704},
+doi = {https://doi.org/10.1016/j.cnsns.2024.107816},
+url = {https://www.sciencedirect.com/science/article/pii/S1007570424000029},
```

Comment:
I would suggest to take the url with `doi.org`, is should be more permanent (well...)